src/dyn/libdyn.cpp

```diff
@@ -244,6 +244,7 @@ void export_dyn_variables_objects(){
       .def_readwrite("tcnbra_thermostats", &dyn_variables::tcnbra_thermostats)
       .def_readwrite("tcnbra_ekin", &dyn_variables::tcnbra_ekin)
       .def_readwrite("qtsh_vars_status", &dyn_variables::qtsh_vars_status)
+      .def_readwrite("kcrpmd_vars_status", &dyn_variables::kcrpmd_vars_status)

```

Comment:
you'll need to add this variable to the `dyn_variables` class, to be able to expose it;

Reply:
The kcrpmd_vars_status variable was added to the dyn_variables files in the previous pull requests.